cmake/linux/package_linux.sh.in

```diff
@@ -40,8 +39,15 @@ function skipped {
 	echo -e "   ${PLAIN}[${YELLOW}skipped${PLAIN}] ${1}"
 }
 
+# Run a command silently, but print output if it fails
+function try_quiet {
+    output="$("$@" 2>&1)" && return
+    echo "$output"
+    return 1
+}
+
 # Blindly assume system arch is appimage arch
-ARCH=$(arch)
+ARCH=$(uname -m)
```

Comment:
I did a quick check, and Ubuntu and Fedora return identical values for `uname -m` and `arch`, so I don't see any regressions caused by this.